package-lock.json

```diff
@@ -2671,9 +2671,9 @@
       }
     },
     "abab": {
-      "version": "1.0.4",
-      "resolved": "https://registry.npmjs.org/abab/-/abab-1.0.4.tgz",
-      "integrity": "sha1-X6rZwsB/YN12dw9xzwJbYqY8/U4=",
+      "version": "2.0.0",
```

Comment:
I'm confused...
Are you using `npm` or `yarn` ?
Having two locks is counterintuitive (as they don't have the same content...